client/my-sites/checkout/composite-checkout/types/transaction-endpoint.ts

```diff
@@ -130,6 +137,34 @@ export function createTransactionEndpointCartFromLineItems( {
 	};
 }
 
+function createTransactionEndpointCartItemFromLineItem(
+	item: WPCOMCartItem
+): WPCOMTransactionEndpointCartItem {
+	return {
+		product_id: item.wpcom_meta?.product_id,
+		meta: item.wpcom_meta?.meta,
+		currency: item.amount.currency,
+		volume: item.wpcom_meta?.volume ?? 1,
+		extra: item.wpcom_meta?.extra,
+	} as WPCOMTransactionEndpointCartItem;
+}
+
+function addRegistrationDataToGSuiteItem(
+	item: WPCOMCartItem,
+	contactDetails: WPCOMTransactionEndpointDomainDetails
+): WPCOMCartItem {
+	if ( ! item.wpcom_meta?.extra.google_apps_users ) {
+		return item;
+	}
```

Comment:
> isGSuiteProductSlug 

Thanks! Good idea.

> Moreover, we actually don't need to provide google_apps_registration_data when a user purchases extra licenses for an existing G Suite account.

Is it bad if we pass that data anyway? It would avoid us having to create a special exception in the transaction for extra licenses.

Reply:
It's bad in the sense that the browser will send a payload that is not required, and that it will make our code confusing to future developers (interestingly that's probably the reason our legacy code in the Store product is designed that way). `isGSuiteProductSlug()` will return `false` for extra licenses so unless I'm missing something we don't have to introduce any additional exception.